metadata.go

```diff
@@ -80,12 +82,36 @@ var debianLicenseText = map[string]string{
 `,
 }
 
+var githubRegexp = regexp.MustCompile(`github\.com/([\w._-]+/[\w._-]+)`)
+
+// regex match github.com/([^/]+/[^/]+) on reply of https://gopkg/?go-get=1
```

Comment:
Remove the comment, it just re-iterates implementation details. Having one place to change is better than having to change two places :)